app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -64,6 +70,17 @@ private void setupLists(List<Message> messageList) {
         for (Message message : messageList) {
             addMessage(message);
         }
+    @Override
+    public int getItemViewType(int position) {
+        if (items.get(position) instanceof MessageHeaderParent)
+            return VIEWTYPE_MESSAGE_HEADER;
+        else if (items.get(position) instanceof Message)
```

Comment:
No difference this function I created because this items list is being used in [ZulipActivity.java](https://github.com/kunall17/zulip-android/blob/patch-9-recyclerview/app/src/main/java/com/zulip/android/activities/ZulipActivity.java#L214)